compliance/unapproved_instance_types/README.md

```diff
@@ -0,0 +1,38 @@
+## Unapproved instance types
+ 
+### What it does
+
+This policy Report on any instances that are running using instance types that are not approved. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/compliance/approved_instance_types) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more.
+
+### Functional Details
+ 
+The policy leverages the RightScale APIs to check instances across all supported clouds. When a non-approved instance type is detected, a report is emailed and the user can choose to Stop the instance after manual approval.
+
+ 
+### Input Parameters
+ 
+- *Email addresses of the recipients you wish to notify* - A list of email addresses to notify
```

Comment:
the  input parameters should match the label names since that is what the user will see and identify.

Reply:
Thanks, I have made the required changes.